### PR TITLE
Adding file submission worker

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

[FLA-71](https://justice.gov.bc.ca/jira/browse/FLA-71)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Run the app

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/file-submission-worker/src/main/resources/application-splunk.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+logging:
+  config:
+    "classpath:logback-splunk.xml"
+
```
Comment (akroon3r) [inline src/backend/file-submission-worker/src/main/resources/application-splunk.yml:4]: GASP

File: README.md [hunk 2]
```diff
@@ -20,11 +21,12 @@ Generic File Submission API (to be used by the Family Law Act Application at fir
...
-| Name             | description                                  | doc                                             |
-| ---------------- | -------------------------------------------- | ----------------------------------------------- |
-| backend          | all server side services                     | [README](src/backend/README.md) |
-| jag-efiling-api  | the main api for interating with the service | [README](src/backend/jag-efiling-api/README.md) |
-| efiling-frontend | the frontend for uploading documents         | [README](src/efiling-frontend/README.md)        |
+| Name                    | description                                  | doc                                             |
+| ----------------        | -------------------------------------------- | ----------------------------------------------- |
+| backend                 | all server side services                     | [README](src/backend/README.md) |
+| jag-efiling-api         | the main api for interating with the service | [README](src/backend/jag-efiling-api/README.md) |
+| file-submission-worker  | the main api for interating with the service | [README](src/backend/file-submission-worker/README.md) |
```
Comment (sdevalapurkar-bcgov) [inline README.md:30]: ### Question
The description for this is same as for `jag-efiling-api`, doesn't seem like this should be the case. Can we provide a better description for this worker here?

File: src/backend/file-submission-worker/src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+server:
+  port: ${PORT:8888}
```
Comment (akroon3r) [inline src/backend/file-submission-worker/src/main/resources/application.yml:2]: DOUBLE GASP

**Comment by akroon3r** [comment]
Merge conflicts, mb

**Pull request merged**